### PR TITLE
I am feeling Lucky

### PR DESCRIPTION
**Pull request by devendrawednesday**

### CODE REVIEW

File: app/containers/HomeContainer/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { connect } from 'react-redux';
 import { createStructuredSelector } from 'reselect';
 import { compose } from 'redux';
 import _ from 'lodash';
+import get from 'lodash/get';
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: this is a double import. 

so basically if shouldn't be importing lodash twice. 


so let's do this. 

check the usages of `_` to find out which are the different methods we are using from lodash
then remove 
```
import _ from 'lodash'
```

and replace with something like this 
```
import get from 'lodash/get'
import debounce from 'lodash/debounce'
```

File: app/containers/HomeContainer/index.js [hunk 2]
```diff
@@ -74,6 +73,22 @@ export function HomeContainer({
   };
   const debouncedHandleOnChange = _.debounce(handleOnChange, 200);
 
+  const activateLasers = () => {
+    // console.log('hello');
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: remove this

File: app/containers/HomeContainer/index.js [hunk 3]
```diff
@@ -74,6 +73,22 @@ export function HomeContainer({
   };
   const debouncedHandleOnChange = _.debounce(handleOnChange, 200);
 
+  const activateLasers = () => {
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: why is this called activateLasers?

File: app/containers/HomeContainer/index.js [hunk 4]
```diff
@@ -74,6 +73,22 @@ export function HomeContainer({
...
+    // console.log('hello');
+    return (
+      <Search
+        data-testid="search-bar"
+        defaultValue={repoName}
+        type="text"
+        onSearch={searchText => {
+          // console.log('!313232'+ searchText);
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: remove this

Comment (alichherawalla) [reply app/containers/HomeContainer/index.js]: there is a shorthand for such stuff. 


what you can do is use something like this 
```
onSearch={debouncedHandleOnChange}
```
it means that whatever the variables you get in the onSearch callback should be passed as is to `debouncedHandleOnChange`

File: app/containers/HomeContainer/index.js [hunk 5]
```diff
@@ -91,13 +106,23 @@ export function HomeContainer({
                 <T id="matching_repos" values={{ totalCount }} />
               </div>
             )}
-            {items.map((item, index) => (
+            {/* {items.map((item, index) => (
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: delete this. we shouldn't have any commented-out unused code in the repo

File: app/containers/HomeContainer/index.js [hunk 6]
```diff
@@ -144,6 +169,27 @@ export function HomeContainer({
           onSearch={searchText => debouncedHandleOnChange(searchText)}
         />
       </CustomCard>
+
+      <CustomCard
+        title={intl.formatMessage({ id: 'repo_search' })}
+        maxwidth={500}
+      >
+        <Text marginBottom={10} id="get_repo_details" />
+        <Search
```
Comment (alichherawalla) [inline app/containers/HomeContainer/index.js]: i see the same code is used somewhere else. please get rid of this duplication

File: package-lock.json [hunk 7]
```diff
@@ -10514,12 +10514,6 @@
       "integrity": "sha1-wguW2MYXdIqvHBYCF2DNJ/y4y3U=",
```
Comment (alichherawalla) [inline package-lock.json]: do not commit the package-lock.json

we already have yarn.lock here

File: package.json [hunk 8]
```diff
@@ -149,15 +149,15 @@
...
-    "eslint-plugin-node": "10.0.0",
     "eslint-plugin-jsx-a11y": "6.2.1",
+    "eslint-plugin-node": "10.0.0",
     "eslint-plugin-prettier": "3.1.1",
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-native": "3.7.0",
     "eslint-plugin-react-hooks": "2.3.0",
-    "eslint-plugin-standard": "4.0.1",
+    "eslint-plugin-react-native": "3.7.0",
```
Comment (alichherawalla) [inline package.json]: why should not need `eslint-plugin-react-native` here

File: app/containers/HomeContainer/index.js (hunk 6 above)
Comment (devendrawednesday) [reply app/containers/HomeContainer/index.js]: i remove this.

File: package-lock.json (hunk 7 above)
Comment (devendrawednesday) [reply package-lock.json]: ok

File: app/containers/HomeContainer/index.js (hunk 5 above)
Comment (devendrawednesday) [reply app/containers/HomeContainer/index.js]: i resolve that.

File: app/containers/HomeContainer/index.js (hunk 1 above)
Comment (devendrawednesday) [reply app/containers/HomeContainer/index.js]: Now i will used single in place of multiple lodash.

File: app/containers/HomeContainer/index.js (hunk 4 above)
Comment (devendrawednesday) [reply app/containers/HomeContainer/index.js]: I remove this.

File: app/containers/HomeContainer/index.js (hunk 2 above)
Comment (devendrawednesday) [reply app/containers/HomeContainer/index.js]: OkI remove this

**Pull request closed without merging**